java-bookstore/src/main/resources/application.properties

```diff
@@ -10,8 +10,11 @@ server.error.whitelabel.enabled=false
 spring.h2.console.enabled=true
 spring.h2.console.path=/h2-console
 
```

Comment:
None of your data seems to be getting seeded! You might be missing something here in application.properties. Also, it doesn't look like you're using the data.sql file that was provided!